src/views/EdgeApplicationsRulesEngine/FormFields/FormFieldsEdgeApplicationsRulesEngine.vue

```diff
@@ -46,6 +46,10 @@
       type: Object,
       required: false,
       default: () => {}
+    },
+    showRequiresApplicationAcceleratorInDescription: {
+      type: Boolean,
+      required: true
```

Comment:
sugestão: por ser um booleano, removeria o 'required' pra evitar warnings desnecessários, uma vez q o padrão é não ter a prop

```suggestion
      type: Boolean,
```